README.md

```diff
@@ -1,47 +1,73 @@
-# ROS/ROS2 bridge for CARLA simulator
- This ROS package is a bridge that enables two-way communication between ROS and CARLA. The information from the CARLA server is translated to ROS topics. In the same way, the messages sent between nodes in ROS get translated to commands to be applied in CARLA. We use [FoxGlove](https://foxglove.dev/) to visualize simulation data.
+The ROS bridge package is a bridge that enables two-way communication between ROS and CARLA. The information from the CARLA server is translated to ROS topics and thus can be visualized in various visualization tools e.g. [FoxGlove Studio](https://github.com/foxglove/studio). This bridge can be used to setup and monitor a CARLA environment for simulated dataset generation for perception systems, image synthesis, and automatic ground truth data generation or testing the trained models.
 
 
-**This version requires CARLA 0.9.12**
+<img src="docs/images/layout.png" width="800"/>

```

Comment:
Do you mind to center the image? -- example https://gist.github.com/DavidWells/7d2e0e1bc78f4ac59a123ddf8b74932d